docs/topics/db/models.txt

```diff
@@ -224,6 +224,15 @@ ones:
     object. If callable it will be called every time a new object is
     created.
 
+:attr:`~Field.db_default`
+    The database-computed default value for the field. This can be a literal value or a
+    database function.
```

Comment:
```suggestion
    The database-computed default value for the field. This can be a literal 
    value or a database function.
```
docs are wrapped at 80 characters